From 89577b59a9b26608fe2de6c9a525d01c657798c0 Mon Sep 17 00:00:00 2001
From: PeterHo-wiwynn <Peter_MH_Ho@wiwynn.com>
Date: Fri, 10 May 2024 16:24:38 +0800
Subject: [PATCH 09/11] platform-mc: fix up tid_t to pldm_tid_t conversions

An earlier patch in this sequence had defined a typedef `tid_t`
at one point in its history and some later patches rely on it.
---
 platform-mc/event_manager.cpp | 2 +-
 platform-mc/event_manager.hpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/platform-mc/event_manager.cpp b/platform-mc/event_manager.cpp
index 7bfb1c8..ff7aa24 100644
--- a/platform-mc/event_manager.cpp
+++ b/platform-mc/event_manager.cpp
@@ -84,7 +84,7 @@ int decode_pldm_cper_event_data(const uint8_t* event_data,
 }
 
 bool EventManager::checkMetaIana(
-    tid_t tid, const std::map<std::string, MctpEndpoint>& configurations)
+    pldm_tid_t tid, const std::map<std::string, MctpEndpoint>& configurations)
 {
     for (const auto& [configDbusPath, mctpEndpoint] : configurations)
     {
diff --git a/platform-mc/event_manager.hpp b/platform-mc/event_manager.hpp
index a3fde89..49f69ab 100644
--- a/platform-mc/event_manager.hpp
+++ b/platform-mc/event_manager.hpp
@@ -113,7 +113,7 @@ class EventManager
                                  size_t eventDataSize);
 
     bool checkMetaIana(
-        tid_t tid, const std::map<std::string, MctpEndpoint>& configurations);
+        pldm_tid_t tid, const std::map<std::string, MctpEndpoint>& configurations);
 
     int createCperDumpEntry(const std::string& dataType,
                             const std::string& dataPath);
-- 
2.25.1

